Include GIT_INFO in source tarball

**Bug Description:**
```
/home/dreamer/Sources/_audio/_plugins/OB-Xf/Builds/Release/geninclude/version_information.cpp:17:21: error: static assertion failed: GIT_COMMIT_HASH not defined. Run cmake/git-version-functions.cmake commands to use this library
   17 | static_assert(""[0] != '\0',
      |               ~~~~~~^~~~~~~
```

Using the exact build instructions from README.md

**Computer Information (please complete the following!):**
 - OS: Debian 12/stable
 - Cmake: 3.31.6
 - GCC: 12.2.0

--- a/cmake/get_dependencies.cmake
+++ b/cmake/get_dependencies.cmake
@@ -65,6 +65,8 @@
 
     # Fetch order means the main cmake runs before git so ugh
     configure_file(${sstplugininfra_SOURCE_DIR}/src/version_information_in.cpp ${CMAKE_BINARY_DIR}/obxfi/version_information.cpp)
+    file(COPY ${CMAKE_BINARY_DIR}/obxfi/version_information.cpp
+            DESTINATION ${CMAKE_BINARY_DIR}/geninclude/)
     add_library(obxf_version_information STATIC)
     target_sources(obxf_version_information PRIVATE  ${CMAKE_BINARY_DIR}/obxfi/version_information.cpp)
     target_include_directories(obxf_version_information PUBLIC ${sstplugininfra_SOURCE_DIR}/include)

